eng/Directory.Build.Common.targets

```diff
@@ -170,7 +170,7 @@
 
   <ItemGroup Condition="'$(UseProjectReferenceToAzureClients)' == 'true'">
     <!-- Find all non-management projects -->
-    <AzureProjects Include="$(RepoRoot)/sdk/**/src/*.csproj" Exclude="$(RepoRoot)/sdk/**/src/*.Management.*.csproj" ProjectPath="%(Identity)" />
+    <AzureProjects Include="$(RepoRoot)/sdk/**/src/*.csproj" Exclude="$(RepoRoot)/sdk/**/src/Azure.ResourceManager*.csproj" ProjectPath="%(Identity)" />
```

Comment:
I don't think this change is what you want.   We should be testing the Resource Manager packages with project references, shouldn't we?

Reply:
I agree we should just remove this Exclude now

Reply:
Remove this Exclude